.storybook/recipes/BaselineCard/BaselineCard.tsx

```diff
@@ -78,8 +78,8 @@ export const BaselineCard = ({
    * Initialize refs to use on the card container and the link
    *  (for clickable card)
    */
-  const cardRef = useRef() as MutableRefObject<HTMLDivElement>;
-  const linkRef = useRef() as MutableRefObject<HTMLAnchorElement>;
```

Comment:
Checking my understanding, `MutableRefObject` is for when you want to manipulate the value of `myref.current` yourself. Otherwise, if the code is just assigning `ref={myref}` then you're fine without it?

Reply:
Yes, that's right.

Although normally I wouldn't expect to manually give a ref its type (since we create them through `useRef` or `createRef`.

But I guess for function params or something we'd have to think about it.